scipy/signal/spectral.py

```diff
@@ -952,3 +951,40 @@ def _fft_helper(x, win, detrend_func, nperseg, noverlap, nfft):
     result = fftpack.fft(result, n=nfft)
 
     return result
+
+def _triage_segments(window, nperseg):
+    """
+    private function that handles defaults.
+    called by _spectral_helper, and also by spectrogram because it requires a
```

Comment:
Clean up the definition line, and add proper capitalization, etc. (see how other helpers are formatted)